src/bench_internal.c

```diff
@@ -282,6 +283,22 @@ void bench_rfc6979_hmac_sha256(void* arg) {
     }
 }
 
+void bench_context_verify(void* arg) {
+    (void)arg;
+    int i;
+    for (i = 0; i < 20; i++) {
+        secp256k1_context_destroy(secp256k1_context_create(SECP256K1_CONTEXT_VERIFY));
+    }
+}
+
+void bench_context_sign(void* arg) {
+    (void)arg;
+    int i;
+    for (i = 0; i < 200; i++) {
+        secp256k1_context_destroy(secp256k1_context_create(SECP256K1_CONTEXT_SIGN));
+    }
+}
```

Comment:
For posterity, it should probably be mentioned this PR also adds 2 new benchmarks.